graphql_api/types/owner/owner.py

```diff
@@ -274,6 +274,19 @@ def resolve_owner_account(owner: Owner, info) -> dict:
     return Account.objects.filter(pk=account_id).first()
 
 
+@owner_bindable.field("isUserOktaAuthenticated")
+@sync_to_async
+@require_part_of_org
+def resolve_is_user_okta_authenticated(owner: Owner, info) -> bool:
+    okta_signed_in_accounts = info.context["request"].session.get(
+        "okta_signed_in_accounts",
```

Comment:
The session key value is incorrect. I think it's actually `okta_logged_in_accounts` (the variable is `OKTA_SIGNED_IN_ACCOUNTS`, which is bad on my part). 

I would suggest to import the variable rather than hard-coding the session key here to avoid the mismatch in different parts of the code: https://github.com/codecov/codecov-api/blob/58faac07fae36e9738f8ef0901fcf718101af67b/codecov_auth/views/okta_cloud.py#L17

(also, feel free to update the variable name or the value stored in the session, as it's inconsistent and I missed it).